README.md

```diff
@@ -1,35 +1,22 @@
 # live-app-versions
 
+### android-live-app-versions
+
+Retrieves information about releases of the [Android Live App](https://github.com/guardian/android-news-app) from the [Play Developer API](https://developers.google.com/android-publisher#publishing).
+
+### ios-live-app-versions
+
 Retrieves information about beta releases of the [iOS Live App](https://github.com/guardian/ios-live) from the 
 [App Store Connect API](https://developer.apple.com/app-store-connect/api/) and stores the results in S3.
 
-## Example Response
-
-Note that this response only ever includes beta builds which have been released to external testers:
-
-```
-{
```

Comment:
This was quite out of date. The API is public - if users really want to see an example response they can just visit the link below.